src/common/data/models/interfaces/RecipientDescribing.ts

```diff
@@ -62,15 +62,15 @@ export function makeContactRecipientDescription(
   return {
     id: data.id,
     kind: RecipientKind.CONTACT,
-    displayedName: data.contactName,
-    walletName: data.contactsWalletName,
-    avatarImageSource: null,
+    displayedName: data.contactName || data.displayedName,
+    walletName: data.contactsWalletName || data.walletName,
```

Comment:
Same applies for walletName... lets see if we can normalise this in the next PR.

Reply:
Same as above this object is referring to 2 flows so, I need to handle this